controller/planner_backlog_blackbox_test.go

```diff
@@ -87,7 +86,7 @@ func (rest *TestPlannerBacklogBlackboxREST) setupPlannerBacklogWorkItems() (test
 			Name:       "Parent Iteration",
 			SpaceID:    testSpace.ID,
 			State:      iteration.IterationStateNew,
-			UserActive: &userActive,
+			UserActive: false,
```

Comment:
Remove this line, since `UserActive` will automatically be `false`.

Reply:
removed it here 237c8ad